types/config.d.ts

```diff
@@ -134,6 +134,7 @@ declare module Config {
     auditRefs: AuditRefJson[];
     description?: string | IcuMessage;
     manualDescription?: string | IcuMessage;
+    supportedModes?:  Gatherer.GatherMode[];
```

Comment:
Required in v9?

Reply:
meh, leaving as the default to always run seems fine even in v9, but I'll leave it up to you